pkg/network/protocols/postgres/telemetry.go

```diff
@@ -15,14 +15,15 @@ import (
 )
 
 const (
+	telemetryBaseBufferSize                 = 160
```

Comment:
Why not using the same value from the kernel?
Having "duplication" can result in having a fork in the future (a.k.a different values)

Reply:
I am starting only with telemetry as the changes in the kernel value are affecting the verifier and are more challenging. I plan to understand better what the value should be, and only then work on the kernel change. I do agree with you that this change can be tested to understand if the fixed size is correct (will start testing on staging). Regarding making it configurable, I am not sure how useful it would be, as when we find the correct size, it should be constant.

Reply:
I have separated it intentionally to distinguish between changing the value and understanding what the most suitable value is. Once I determine the value we plan to use, I will ensure that the values for telemetry and the kernel are equal.

Reply:
then, I don't understand the goal  of the PR
if the branch is used only for testing purposes, you don't need a PR

Reply:
The purpose is to create a method to better understand the most appropriate buffer size. I have created a separate PR with a configuration change to allow for faster and more accurate understanding.

I can create a new image with a different value each time and deploy it to understand the impact. However, I'm not sure if this is better than having one PR, which would allow me to update the configuration more easily.

Reply:
but you can use this branch for the testing until you get the results of the "optimal" buffer size
I'm curious why we need to merge it to main